deploy.yaml

```diff
@@ -50,6 +50,26 @@
         dest: "{{ openshift_files }}/osbs-box.env"
 
 
+- name: Create all projects (namespaces)
+  hosts: osbs_box_host
+  gather_facts: false
+  tags:
+    - namespaces
+    - openshift
```

Comment:
This play is now a prerequisite for all the other plays and would therefore need all the tags. But keeping the tags functional will probably be a nightmare and I'm not sure if anyone other than me has ever used them, so maybe it's fine.

Reply:
Agree with everything you said :grin: